docs/mddocs/Quickstart/install_linux_gpu.md

```diff
@@ -15,9 +17,144 @@ IPEX-LLM currently supports the Ubuntu 20.04 operating system and later, and sup
 
 ## Install Prerequisites
 
-### Install GPU Driver
+### For Intel Core™ Ultra Processors with Intel Arc™ Graphics (a.k.a. Meteor Lake)
 
-#### For Linux kernel 6.2
+> [!NOTE]
+> For IPEX-LLM on Linux with Meteor Lake integrated GPU, we have currently verified on Ubuntu 22.04 with kernel `6.5.0-35-generic`.
+
+#### 1. Check current kernel version
+
+You could check your current kernel version through:
+
+```bash
+uname -r
+```
+If the version displayed is not `6.5.0-35-generic`, downgrade or upgrade the kernel to the recommended version.
+
+#### 2. (Optional) Downgrade / Upgrade to kernel 6.5.0-35
+
+If your current kernel version is not `6.5.0-35-generic`, you could downgrade or upgrade it by:
+
+```bash
+export VERSION="6.5.0-35"
+sudo apt-get install -y linux-headers-$VERSION-generic
+sudo apt-get install -y linux-image-$VERSION-generic
+sudo apt-get install -y linux-modules-extra-$VERSION-generic
+
+sudo sed -i "s/GRUB_DEFAULT=.*/GRUB_DEFAULT=\"1> $(echo $(($(awk -F\' '/menuentry / {print $2}' /boot/grub/grub.cfg \
+| grep -no $VERSION | sed 's/:/\n/g' | head -n 1)-2)))\"/" /etc/default/grub
+
+sudo update-grub
+```
+
+And the reboot your machine:
+
+```bash
+sudo reboot
+```
+
+After rebooting, you can use `uname -r` again to see that your kernel version has been changed to `6.5.0-35-generic`.
+
+#### 3. Enable driver support through `force_probe` flag
+
+Next, you need to enable driver support on kernel `6.5.0-35-generic` through `force_probe` parameter：
+
+```bash
+sudo sed -i "s/GRUB_CMDLINE_LINUX_DEFAULT=\"quiet splash\"/GRUB_CMDLINE_LINUX_DEFAULT=\"quiet splash i915.force_probe=7d55\"/g" /etc/default/grub
```

Comment:
We had better get force probe device id from dmesg from:
```
(base) arda@Xiaoxin03-Ubuntu:~$ sudo dmesg  | grep i915
[    3.820895] i915 0000:00:02.0: Your graphics device 7d55 is not properly supported by i915 in this
               kernel version. To force driver probe anyway, use i915.force_probe=7d55
[   64.598709] sof-audio-pci-intel-mtl 0000:00:1f.3: init of i915 and HDMI codec failed
```

Reply:
Some MTLs' device IDs are 7d40 and 7d45.

Reply:
Have updated :)